Sources/Fuzzilli/Fuzzer.swift

```diff
@@ -15,6 +15,17 @@
 import Foundation
 
 public class Fuzzer {
+    //When importing a corpus, this determines how valid samples are added to the corpus
+    public enum CorpusImportMode {
```

Comment:
Yep, this is great! Can you please also mention how the two modes handle minimization of the samples